.github/workflows/build.yml

```
@@ -130,6 +133,17 @@ jobs:
         with:
           name: Northstar.release.${{ env.NORTHSTAR_VERSION }}
           path: northstar
+      - name: Create zip to upload
+        run: |
+          cd northstar/
+          zip --recurse-paths --quiet Northstar.release.${{ env.NORTHSTAR_VERSION }}.zip .
+          mv Northstar.release.${{ env.NORTHSTAR_VERSION }}.zip ../
+      - name: Upload zip to release draft
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/') && !contains(github.ref, '-rc')
```

Is there a reason why you check ref directly here instead of using `env.NORTHSTAR_VERSION`?

---

The reason is that I copy/pasted existing stuff and didn't properly change it.

I assume `startsWith(github.ref, 'refs/tags/')` should stay the same but `!contains(github.ref, '-rc') ` should be changed to `!contains(${{ env.NORTHSTAR_VERSION }}, '-rc') ` ?

---

If is already an expression; you don't need the `${{` around that.

---

So basically

```suggestion
        if: startsWith(github.ref, 'refs/tags/') && !contains(env.NORTHSTAR_VERSION, '-rc')
```

?

---

Might also be good to check `refs/tags/v` for consistency throughout.

---

> Might also be good to check `refs/tags/v` for consistency throughout.

Done in 077b5913cff1f726aba90dae484627792e5c2268

---

> Yes, that should work, but test it.

Done in 30d9520001c2bb2522b37db61fd8a9124445ce5c tested in

- https://github.com/GeckoEidechse/Northstar/actions/runs/4358004146
- https://github.com/GeckoEidechse/Northstar/actions/runs/4358005608